xarray/core/parallelcompat.py

```diff
@@ -0,0 +1,199 @@
+"""
+The code in this module is an experiment in going from N=1 to N=2 parallel computing frameworks in xarray.
+It could later be used as the basis for a public interface allowing any N frameworks to interoperate with xarray,
+but for now it is just a private experiment.
+"""
+
+from abc import ABC, abstractmethod
+from typing import Any
+
+import numpy as np
+
+from . import indexing, utils
+from .pycompat import DuckArrayModule, is_duck_dask_array
+
+CHUNK_MANAGERS = {}
+
+
+def _get_chunk_manager(name: str) -> "ChunkManager":
+    if name in CHUNK_MANAGERS:
+        chunkmanager = CHUNK_MANAGERS[name]
+        return chunkmanager()
+    else:
+        raise ImportError(f"ChunkManager {name} has not been defined")
+
+
+class ChunkManager(ABC):
```

Comment:
```suggestion
class ParallelBackend(ABC):
```
?

Reply:
I just tried to avoid the word "backend" because we use it for the file I/O, our only other entrypoint...